server/src/test/java/io/crate/integrationtests/CreateTableIntegrationTest.java

```diff
@@ -227,4 +226,14 @@ tn boolean as (ta = false)
         execute("select * from t");
         assertThat(response.cols()).isEqualTo(new String[] {"ta", "tb", "tc", "ti", "tm", "tn"});
     }
+
+    @Test
+    public void testCreateTableWithTotalFieldsLimit() {
```

Comment:
I'd tend to put it into `TableSettingsTest` next to `testMappingTotalFieldsLimit`

Reply:
Thx! will do the same in the 5.10 PR